src/Components/Modals/ManageColumnsModal.js

```diff
@@ -54,58 +57,67 @@ export const ManageColumnsModal = ({ modalColumns, isModalOpen, setModalOpen, sa
             title='Manage columns'
             onClose={handleCancel}
             isOpen={isModalOpen}
-            variant='small'
-            description={
-                <TextContent>
-                    <Text component={TextVariants.p}>Selected categories will be displayed in the table</Text>
-                    <Split hasGutter>
-                        <SplitItem>
-                            <Button isInline onClick={handleSelectAll} variant="link">
-                                Select all
-                            </Button>
-                        </SplitItem>
-                        <SplitItem>
-                            <Button isInline onClick={handleResetToDefault} variant="link">
-                                Reset to default
-                            </Button>
-                        </SplitItem>
-
-                    </Split>
-                </TextContent>
-            }
-            actions={[
+            variant={ModalVariant.small}
+            aria-labelledby="select-columns-modal-title"
+            aria-describedby="modal-box-select-columns-form"
+        >
+            <ModalHeader
+                title="Manage columns"
+                descriptorId="modal-box-select-columns-form"
+                labelId="select-columns-modal-title"
+                description={
+                    <Content>
+                        <Content component={ContentVariants.p}>Selected categories will be displayed in the table</Content>
```

Comment:
Do we need the extra wrapper here? Is it possible to simplify it?
Since the description could also be directly part of the modal header content itself, I think we can directly use the description prop in the `ModalHeader`, rather than wrapping the description in a `Content` component - [reference](https://www.patternfly.org/components/modal/#modalheader)